manifests/v1beta1/installs/katib-openshift/katib-config-patch.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
```

Comment:
This file should not be used anymore and as the image tags have been added to the controller deployment [here](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/components/controller/katib-config.yaml) instead.